src/Test/TestDumbMath.java

```diff
@@ -0,0 +1,98 @@
+package Test;
+
+import CommandingOfficers.Commander;
+import Engine.Army;
+import Engine.GameInstance;
+import Engine.GameScenario;
+import Engine.Combat.CombatContext;
+import Engine.Combat.StrikeParams.BattleParams;
+import Engine.Combat.CombatContext.CalcType;
+import Terrain.MapInfo;
+import Terrain.MapLibrary;
+import Terrain.MapMaster;
+import Units.UnitContext;
+import Units.UnitModel;
+
+/**
+ * Tests Power of Money variants because I make poor life choices
+ */
+public class TestDumbMath extends TestCase
```

Comment:
Setting aside the issue of the quality of your life choices, I think I'd prefer a different name for this class.

Reply:
On the other hand, I guess it will technically only ever matter if a test fails, and then context will be there to save the day, but I mean, still.

Reply:
I was thinking I might eventually put more things in it, but I guess Javier already exists and has his own file.
Thoughts on a "cart math" test file for these purposes?
...though that would technically only cover *one* Javier...

Naming is hard.

Reply:
One of the two hard problems, yeah.

This file is specific to Collin and his money-based buffs, so I'd prefer to name it for that, and have a separate file for verifying Javier functionality (which you have). A global "cart math" file is probably too general.